sass/includes/_record-links.scss

```diff
@@ -0,0 +1,92 @@
+.record-links {
+    &__wrapper {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        padding-left:10%;
```

Comment:
And lets put this left & right 10% padding in a larger breakpoint so it's only targeting tablet/desktop